web-admin/src/routes/github/connect/+page.svelte

```diff
@@ -24,13 +20,19 @@
     } else {
       user = data.user;
     }
+
+    const urlParams = new URLSearchParams(window.location.search);
+    let autoredirect = urlParams.get("auto_redirect");
+    if (autoredirect === "true") {
+      handleGoToGithub();
+    }
```

Comment:
Why not just open the browser at `ADMIN_URL/github/connect` directly instead of having the `auto_redirect` query parameter?

Reply:
I wasn't sure if same UI pages are being/will be used in UI flow where auto redirects may not make sense. I can change it redirect always as well if that's not the case.

Reply:
Sorry, I didn't mean to always redirect. I meant that the CLI could just directly open `ADMIN_URL/github/connect` instead of going to `FRONTEND_URL/github/connect` to get redirected (avoids a redirect and a page flash)